lib/unsanitized.js

```diff
@@ -0,0 +1,83 @@
+// names of escaping functions that we acknowledge
+const VALID_ESCAPERS = ["Sanitizer.escapeHTML", "escapeHTML"];
+const VALID_UNWRAPPERS = ["Sanitizer.unwrapSafeHTML", "unwrapSafeHTML"];
+
+class Unsanitized {
```

Comment:
Checks in checks.js. checks.allowedExpression() reads nicely. `checks = new Unescape(context)` only so-and- so.
Having said that, allowedExpression should be refactored into isAllowedExpression but that can be a follow-up.

To re-emphasize: I agree that working on the context and thus configurization within that class makes sense, I just did not understand the implications. Keep that!